code/hud/hudconfig.cpp

```diff
@@ -349,6 +288,13 @@ struct HC_gauge_region	HC_gauge_regions[GR_NUM_RESOLUTIONS][NUM_HUD_GAUGES] =
 	}
 };
 
+struct gauge_coords {
```

Comment:
Please doxy

Reply:
I have doxy'd all the things from here on.